internal/zinc-persist/src/main/scala/sbt/internal/inc/FileAnalysisStore.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Zinc - The incremental compiler for Scala.
+ * Copyright 2011 - 2017, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * This software is released under the terms written in LICENSE.
+ */
+
+package sbt
+package internal
+package inc
+
+import java.io._
+import java.util.Optional
+import java.util.zip.{ ZipEntry, ZipInputStream }
+
+import com.google.protobuf.{ CodedInputStream, CodedOutputStream }
+import sbt.inc.ReadWriteMappers
+import sbt.io.{ IO, Using }
+import xsbti.compile.{ AnalysisContents, AnalysisStore }
+
+import scala.util.control.Exception.allCatch
+
+object FileAnalysisStore {
+  private final val analysisFileName = "inc_compile.txt"
+  private final val companionsFileName = "api_companions.txt"
```

Comment:
True. Those are never exposed, they are only internal to the zip file though. Do you prefer the `.bin` extension?